apps/passport-server/test/pretix/mockDevconnectPretixApi.ts

```diff
@@ -16,9 +12,6 @@ export function getDevconnectMockPretixAPIServer(
     handlers.push(
       rest.get(orgUrl + "/events", (req, res, ctx) => {
         const org = mocker.getOrgByUrl(orgUrl);
-        if (getAuthToken(req) !== org.token) {
-          return res(ctx.status(403), ctx.text("Invalid token"));
-        }
```

Comment:
Perhaps we should make the issue now and then add a `// TODO:` comment here referencing the issue.

Reply:
I've added an issue here: https://github.com/proofcarryingdata/zupass/issues/1004

I don't think the comment is needed, we will prioritize the issue at next Monday's backlog grooming.